vispy/visuals/axis.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, pos=None, domain=(0., 1.), tick_direction=(-1., 0.),
                  scale_type="linear", axis_color=(1, 1, 1),
                  tick_color=(0.7, 0.7, 0.7), text_color='w',
                  minor_tick_length=5, major_tick_length=10,
-                 tick_width=2, tick_label_margin=5, tick_font_size=8,
+                 tick_width=2, tick_label_margin=12, tick_font_size=8,
```

Comment:
Was this intended for this PR?

Reply:
i do think 12 should be the default (since the `major_tick_length` defaults to 10, I think the labels will almost always [be on top of the ticks](https://github.com/vispy/vispy/issues/1742#issuecomment-549382586) with a margin of 5).  but the change is a bit out of place here.  what do you think?  leave it in, or consider a different PR?

Reply:
I am OK with this staying here. There is a very good chance people are already changing this themselves to make it look better.